src/label_projection/metrics/accuracy/config.vsh.yaml

```diff
@@ -0,0 +1,42 @@
+functionality:
+  name: "accuracy"
```

Comment:
Please think about whether accuracy & F1 should be combined into one metric component that computes both F1 and accuracy. It's definitely OK to decide to keep them separate :)